src/Core/src/Platform/Android/MauiApplication.cs

```diff
@@ -37,15 +37,15 @@ public override void OnCreate()
 
 			var rootContext = new MauiContext(mauiApp.Services, this);
 
-			var applicationContext = rootContext.MakeApplicationScope(this);
+			rootContext.InitializeApplicationScope(this);
```

Comment:
For all the platforms, we no longer need to wrap the services to just add the app. We can add it using the `PlatformApplicationProvider` and related code.  See the new file: src/Core/src/Hosting/MauiAppBuilderExtensions.cs

Reply:
Gonna revert this for now to reduce these changes...